.xrefcheck.yaml

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2022 Tocqueville Group
```

Comment:
Apparently, the CI step is not pulling the xrefcheck version from sources.json...

I guess the command needs to be changed to `nix run -f . -c xrefcheck`

Also, please get rid of `soft_fail: true` here (and in other repos too)

https://github.com/tqtezos/stablecoin/blob/5d43f194b11d3fcb2510ed6e508f3d0a8cde5aca/.buildkite/pipeline.yml#L58-L61

Reply:
fixed - actually had to define it in the `default.nix` as well.